extensions/github1s/package.json

```diff
@@ -298,12 +298,6 @@
         "category": "GitHub1s",
         "icon": "$(globe)",
         "enablement": "github1s:adapters:default:platformName != 'GitHub' && github1s:adapters:default:platformName != 'GitLab' && github1s:adapters:default:platformName != 'Bitbucket' && github1s:adapters:default:platformName != 'npm'"
-      },
-      {
-        "command": "github1s.commands.syncSourcegraphRepository",
-        "title": "Sync Sourcegraph Repository",
-        "category": "GitHub1s",
-        "enablement": "resourceScheme =~ /^(github1s|gitlab1s|bitbucket1s)/"
```

Comment:
Any reason to this one?

Reply:
I added a button at the lower left corner of the UI to trigger this command. In some cases, there may be a delay in the file updates of Sourcegraph, and this command is used to trigger the synchronization of Sourcegraph, which is experimental in nature.

Reply:
Repo search doesn't seem to be working:

<img width="908" alt="image" src="https://github.com/user-attachments/assets/0f07a033-917b-437e-b43f-7e9e06a8b0b0" />


However, it can search from opened files:

<img width="888" alt="image" src="https://github.com/user-attachments/assets/5bc92604-ec4a-4a55-954d-502357268222" />

Reply:
The search capability is provided by Sourcegraph, it seems they haven't indexed this repository. Other more repositories can be searched normally.